cli/src/commands/generate/store.ts

```diff
@@ -42,7 +42,7 @@ export default class GenerateStore extends Command {
     }
 
     await cloneGitRepository({
-      projectDir,
+      projectDir: projectDir.concat('/backend'),
```

Comment:
There is a check (potentially, could be more checks), that we're missing, having the concat method NOT chained to the `path.resolve(projectName)`

line:29
```
const projectDir = path.resolve(projectName);
```
followed by line:31

```
if (await existsDirectory(projectDir)) { ... }
```

.concat('/backend') should be chained to path.resolve(projectName) (line:29)

```suggestion
      projectDir,
```